docroot/themes/custom/uids_base/templates/content/node--person--teaser.html.twig

```diff
@@ -88,6 +88,12 @@
   {% set headline_level = heading_size %}
 {% endif %}
 
+{% if '1' in node.field_person_website_link_direct.0.value and node.field_person_website.0.uri %}
+  {% set node_link = node.field_person_website.0.uri %}
+{% else %}
+  {% set node_link = url %}
+{% endif %}
```

Comment:
Wondering if there was a reason to do this in the template vs in a preprocess function?

Reply:
@pyrello based on https://github.com/uiowa/uiowa/pull/6053#discussion_r1064710950 can this be resolved?